LICENSE

```diff
@@ -1,202 +1,7 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                           Copyright (c) 2018 ChainSafe Systems
-                           http://www.apache.org/licenses/
+Copyright 2019- ChainSafe Systems
 
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
This is a different license. I went with the simpler MIT license.

Reply:
Lets just continue to use apache2.0 for the time being 😄

Reply:
Ok, I can change it.

Since this is a completely new codebase, this is an opportunity to reevaluate the license we're using, since we're applying a license to virgin code. Are there good reasons not to use MIT?

Reply:
from https://exygy.com/which-license-should-i-use-mit-vs-apache-vs-gpl/

> The Apache license has a similar philosophy to the MIT, but uses more words. The wordiness creates greater specificity about contributors’ obligations, which might help in a dispute.  But it also can be a turnoff — “Do I need to have my lawyer look at this?” comes up more with Apache than MIT. It works well for organizations or projects that are larger and managing more contributors, but don’t care about others commercializing the work. It also can help bring on board organizations that are more concerned about software patents or patent trolls.

the last part is the most important difference for us between Apache and MIT.

Reply:
ok, sounds good, thanks Aidan

Reply:
Anytime, thank you :)